### ISSUE TITLE
Incorrectly Determining Height of Characters

### ISSUE THREAD

**Issue by jeremyschiff** [pdfminer/pdfminer.six#48]
[WrongFontSizes3.pdf](https://github.com/pdfminer/pdfminer.six/files/822650/WrongFontSizes3.pdf)

The following simple python code illustrates a bug with parsing the attached PDF file.  Specifically, it incorrectly determines the height of text.  Namely it thinks the small text is much larger than the big text.

```
from pdfminer.pdfparser import PDFParser
from pdfminer.pdfpage import PDFPage
from pdfminer.pdfdocument import PDFDocument
from pdfminer.pdfinterp import PDFResourceManager, PDFPageInterpreter
from pdfminer.converter import PDFPageAggregator
from pdfminer.layout import LAParams, LTTextBox, LTChar

def parse_pages():

    fp = open('WrongFontSizes3.pdf', 'rb')
    parser = PDFParser(fp)
    doc = PDFDocument(parser)
    parser.set_document(doc)

    rsrcmgr = PDFResourceManager()
    laparams = LAParams(char_margin=3.5, all_texts=True)
    device = PDFPageAggregator(rsrcmgr, laparams=laparams)
    interpreter = PDFPageInterpreter(rsrcmgr, device)
    for page in PDFPage.create_pages(doc):
        interpreter.process_page(page)
        layout = device.get_result()
        yield layout

if __name__ == '__main__':
    for page in parse_pages():
        for tbox in page:
            if not isinstance(tbox, LTTextBox):
                continue
            for line in tbox:
                for char in line:
                    if not isinstance(char, LTChar):
                        continue
                    print char.get_text().encode('UTF-8'), char.size
```
Output:
```
B 29.4555
i 29.4555
g 29.4555
T 29.4555
e 29.4555
x 29.4555
t 29.4555
S 66.96
m 66.96
a 66.96
l 66.96
l 66.96
  66.96
  66.96
T 66.96
e 66.96
x 66.96
t 66.96

Process finished with exit code 0
```

**Comment by LuJyKa** [comment]
you can change code in pdfminer/pdffont.py line 511, class PDFFont
`    def get_height(self):
        # h = self.bbox[3]-self.bbox[1]
        # if h == 0:
        #    h = self.ascent - self.descent
        h = self.ascent - self.descent
        return h * self.vscale`

**Comment by pietermarsman** [comment]
Possibly related to #203

**Comment by pietermarsman** [comment]
I've checked if this bug is introduced recently. The answer is: no, it has at least been there since 7b620b3146c4bd43be83487f036cb8bf9031c790.

**Comment by pietermarsman** [comment]
Other observations:

- If I copy the text from the pdf to microsoft word, it has font-size 22 for the big text and font-size 12 for the small text. These match the pdfminer.six computed.
- The font-height is computed based on the `/FontBBox` property of the font. If I increase the upper y-coordinate drastically the pdfminer.six computed font-sizes also increase. However, Adobe Acrobat Reader displays the font in the same size. This could suggest that `/FontBBox` should not influence the size of the drawn characters.
- Section 5.1.1 and Section 5.3.3 of the PDF Reference only refers to the text font size and text matrix to compute the size and spacing of glyphs. No use of the `/FontBBox`. 

In short: my conclusion is that this bug is caused by multiplying with the font-height (i.e. the height of the `/FontBBox`). This multiplication should be removed.

**Comment by pietermarsman** [comment]
@jeremyschiff do you have time for a review? Could you take a look at #348?
